Component testing broken on next@11.0.2-canary.20

### Current behavior

Using an early release of `next` (`11.0.2-canary.20`) breaks `cypress` component testing. This release is critical to many users as it fixes a performance issue with their `Image` component, blocking usage for many. See https://github.com/vercel/next.js/issues/23637

Running tests with this version throws `Error: Cannot find module 'next/dist/next-server/server/config'`.

### Desired behavior

Not throw an error.

### Test code to reproduce

Running component tests.

### Cypress Version

8.0.0

### Other

```
"@cypress/react": "^5.9.2",
"@cypress/webpack-dev-server": "^1.4.0",
"@testing-library/cypress": "^8.0.0",
"cypress": "^8.0.0",
"html-webpack-plugin": "5",
"webpack": "5",
"webpack-dev-server": "3"
```

This is still erroring with `Cannot find module 'next/dist/next-server/server/config'`? 

Is this update meant to read `next/dist/server/config.js` instead? Can we override this path? It's a major blocker, the project can't run without tests.

--- a/npm/react/plugins/next/findNextWebpackConfig.js
+++ b/npm/react/plugins/next/findNextWebpackConfig.js
@@ -14,9 +14,9 @@ async function getNextWebpackConfig (config) {
       // is not in the next-server folder anymore.
       // @ts-ignore
       loadConfig = require('next/dist/server/config').default
+    } else {
+      throw e
     }
-
-    throw e
   }
   const nextConfig = await loadConfig('development', config.projectRoot)
   const nextWebpackConfig = await getNextJsBaseWebpackConfig(

